cranelift/filetests/filetests/isa/x64/simd-bitwise-compile.clif

```diff
@@ -349,19 +349,20 @@ block0(v0: i8x16, v1: i32):
 ;   Entry block: 0
 ; Block 0:
 ;   (original IR block: block0)
-;   (instruction range: 0 .. 12)
+;   (instruction range: 0 .. 13)
 ;   Inst 0:   pushq   %rbp
 ;   Inst 1:   movq    %rsp, %rbp
 ;   Inst 2:   movdqa  %xmm0, %xmm1
-;   Inst 3:   movdqa  %xmm1, %xmm0
-;   Inst 4:   punpcklbw %xmm0, %xmm0
-;   Inst 5:   psraw   $11, %xmm0
-;   Inst 6:   punpckhbw %xmm1, %xmm1
-;   Inst 7:   psraw   $11, %xmm1
-;   Inst 8:   packsswb %xmm1, %xmm0
-;   Inst 9:   movq    %rbp, %rsp
-;   Inst 10:   popq    %rbp
-;   Inst 11:   ret
+;   Inst 3:   punpcklbw %xmm0, %xmm1
+;   Inst 4:   movdqa  %xmm0, %xmm2
+;   Inst 5:   punpckhbw %xmm0, %xmm2
+;   Inst 6:   psraw   $11, %xmm1
+;   Inst 7:   psraw   $11, %xmm2
+;   Inst 8:   packsswb %xmm2, %xmm1
+;   Inst 9:   movdqa  %xmm1, %xmm0
+;   Inst 10:   movq    %rbp, %rsp
+;   Inst 11:   popq    %rbp
+;   Inst 12:   ret
```

Comment:
Managed to get it down to 12 instructions, instead of 13. Still one additional move compared to before. meh.